components/clientComponents/forms/FormGroup/FormGroup.tsx

```diff
@@ -22,8 +22,6 @@ export const FormGroup = (props: FormGroupProps): React.ReactElement => {
       data-testid="formGroup"
       className={classes}
       aria-describedby={ariaDescribedBy}
-      // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-      tabIndex={0}
```

Comment:
Any idea why this was here to begin with?

Reply:
Good question. On further testing I can see this was used by the form validation skip ahead links. The problem with tabindex=0 is it is then in the tab order which is confusing for assistive technology users because tabbing to a radio group will announce it etc. but the user is not really in the radio group yet. A nice compromise is to use tabindex=-1 so the form group is still programmatically focussable and it is not in the natural tab order.